homeassistant/components/arcam_fmj/device_trigger.py

```diff
@@ -33,7 +36,7 @@
 
 async def async_get_triggers(
     hass: HomeAssistant, device_id: str
-) -> list[dict[str, str]]:
+) -> GetAutomationsResult:
```

Comment:
I would like to revert this, for reasoning and the review see: <https://github.com/home-assistant/core/pull/72163#issuecomment-1133865649>